addons/hr_fleet/models/employee.py

```diff
@@ -17,17 +17,26 @@ class Employee(models.Model):
 
     def action_open_employee_cars(self):
         self.ensure_one()
-        cars = self.env['fleet.vehicle.assignation.log'].search([
-            ('driver_id', 'in', (self.user_id.partner_id | self.sudo().address_home_id).ids)]).mapped('vehicle_id')
-
-        return {
+        cars = self.env['fleet.vehicle.assignation.log'].sudo().search([
+            ('driver_id', 'in', (self.user_id.partner_id | self.sudo().address_home_id).ids)
+        ]).mapped('vehicle_id').sudo(False)
+        response = {
             "type": "ir.actions.act_window",
             "res_model": "fleet.vehicle",
```

Comment:
```suggestion
            "res_model": "fleet.vehicle.assignation.log",
```
We should change the model here and not open vehicles